files/zh-cn/learn/tools_and_testing/client-side_javascript_frameworks/svelte_getting_started/index.md

```diff
@@ -0,0 +1,504 @@
+
+若想知道更多相关信息也可以阅读下列资料：
+
+- 可以至 npm 文档了解[“关于 npm 的信息”](https://docs.npmjs.com/about-npm)
+- 可以至 npm 博客了解[“介绍 npx”](https://blog.npmjs.org/post/162869356040/introducing-npx-an-npm-package-runner)
+
+### 建构你的第一个 Svelte 应用程序
+
+最简单的方式就是使用入门模板来建构初始应用程序。你可以至 [sveltejs/template](https://github.com/sveltejs/template) 下载并解压缩入门模板或者使用 [degit](https://github.com/Rich-Harris/degit) 来做这件事。
+
+为了建构你第一个入门模板应用程序，请先执行下列终端指令：
+
+```bash
+npx degit sveltejs/template moz-todo-svelte
+cd moz-todo-svelte
+npm install
+npm run dev
+```
+
+> **备注：** 不用太惊讶 degit 做了这么多事情——它其实就只是让你可以从 git 仓库下载和解压缩最新版本的内容。使用它比起使用 `git clone` 更快能达到我们的目的，因为它并不会下载整个仓库的历史以及不会创建完整的本地克隆。
+
+之后执行 `npm run dev`，Svelte 将会编译和建构你的应用程序。它会跑在本机 `localhost:8080` 地址。当你对源代码文件做了变动，Svelte 将会侦测到文件更新并自动地重新编译和刷新你的应用程序。你的浏览器将会显示如下图：
+
+![一个简单的起始页，上面写着 hello world，并提供指向官方 svelte 教程的链接](01-svelte-starter-app.png)
+
+### 应用程序结构
+
+入门模板有下列结构：
+
+```plain
+moz-todo-svelte
+├── README.md
+├── package.json
+├── package-lock.json
+├── rollup.config.js
+├── .gitignore
+├── node_modules
+├── public
+│   ├── favicon.png
+│   ├── index.html
+│   ├── global.css
+│   └── build
+│       ├── bundle.css
+│       ├── bundle.js
+│       └── bundle.js.map
+├── scripts
+│   └── setupTypeScript.js
+└── src
+    ├── App.svelte
+    └── main.js
+```
+
+内容解释如下：
+
+- `package.json` 和 `package-lock.json`：Node.js/npm 用它来组织化管理你的项目。在这个教学系列中，你完全不需要了解这个文件，但如果你想要学习更多的话，你可以至 npmjs.com 阅读 [`package.json` 处理](https://docs.npmjs.com/cli/configuring-npm/package-json)；我们在[基础包管理教学系列](/zh-CN/docs/Learn/Tools_and_testing/Understanding_client-side_tools/Package_management)也有谈到。
+- `node_modules`：这里是 node 存放项目依赖的地方。这些依赖在正式环境不会用到，只有开发时才会被使用到。
+- `.gitignore`：告诉 git 有哪些文件或文件夹不要纳入项目版本控制——如果你决定将应用程序包含在 git 仓库中，它还蛮实用的！
+- `rollup.config.js`：Svelte 使用 [rollup.js](https://rollupjs.org/) 做为模块打包工具。这个配置文件告诉 rollup 如何编译和建构你的应用程序。假如你偏好使用 [webpack](https://webpack.js.org/)，你可以改执行 `npx degit sveltejs/template-webpack svelte-app` 来建构你的初始应用程序。
+- `scripts`：包含所需的设置脚本。现在应该只有 `setupTypeScript.js`。
+
+  - `setupTypeScript.js`：此脚本用来支持 Svelte 使用 TypeScript。我们将在之后的文章谈到更多。
+
+- `src`：这个目录主要放你应用程序的源代码——你写的程序代码基本都会放在这。
+
+  - `App.svelte`：在你的应用程序中它是最高层级的组件。不过它现在只会渲染出“Hello World!”的讯息给你看。
+  - `main.js`：我们应用程序的进入点。它会实例化 `App` 组件并将其绑定至我们 HTML 页面的 body 上。
+
+- `public`：这个目录包含所有我们会发布于正式环境的文件。
+
+  - `favicon.png`：这个是你应用程序的图标。现在应该会是 Svelte 的标志。
+  - `index.html`：这个是你应用程序的主页面。最初为一个空的 HTML5 页面，接着会载入由 Svelte 产生的 CSS 文件和 js 包。
+  - `global.css`：这个文件包含没有范围限制的样式。这是一个正规的 CSS 文件，将会套用至整个应用程序。
+  - `build`：这个文件夹会包含那些编译生成的 CSS 和 JavaScript 源代码。
+
+    - `bundle.css`：所有定义于你个别组件中的样式，都会被 Svelte 编译生成至这个 CSS 文件中。
+    - `bundle.js`：所有你的 JavaScript 源代码都会被编译生成至这个 JavaScript 文件中。
+
+## 让我们来看看第一个 Svelte 组件
+
+组件是 Svelte 应用程序建构的基础。组件位于 `.svelte` 文件中，可以使用 HTML 的超集来编写。
+
+总共有三个区块—— `<script>`，`<styles>` 和标记（markup）区块——各区块存在与否由你决定。
```

Comment:
```suggestion
总共有三个区块——`<script>`，`<styles>` 和标记（markup）区块——各区块存在与否由你决定。
```

Reply:
Sorry, I think space is not necessary here, https://github.com/mdn/translated-content/blob/63e67cf315d3bcb18b807e868d91ca9e838730ba/docs/zh-cn/translation-guide.md?plain=1#L124C21-L124C21

<img width="528" alt="image" src="https://github.com/mdn/translated-content/assets/84226578/f12a184a-3278-4ba9-98ac-685070fe462c">

The typesetting routine is to neglect the "`" in typesetting, so this rule apply to this situation as well

Reply:
@yin1999 I am glad to hear your opinion

Reply:
Hey @hanyujie2002. We could remove the space here. For this rule, we could "ignore" the backtick (`` ` ``) and angle brackets here. So we could consider this sentence as:

```md
总共有三个区块—— script，styles 和标记（markup）区块——各区块存在与否由你决定。
```

So we could remove the space before "script".

Reply:
My mistake, I misread it and thought the code reviewer added a space instead of removing the space ＞﹏＜

Reply:
never mind